pkg/collector/corechecks/cluster/ksm/kubernetes_state_transformers.go

```diff
@@ -228,27 +264,18 @@ func containerResourceLimitsTransformer(s sender.Sender, name string, metric ksm
 // submitContainerResourceMetric can be called by container resource metric transformers to submit resource-specific metrics
 // metricSuffix can be either requested or limit
 func submitContainerResourceMetric(s sender.Sender, name string, metric ksmstore.DDMetric, hostname string, tags []string, metricSuffix string) {
-
-	var allowedResources = map[string]string{
```

Comment:
@L3n41c We merged `allowedResources` for containers and nodes. I wonder if I should revert this change as this could add weird resources.